lib/pbench/server/api/resources/users_api.py

```diff
@@ -282,9 +282,9 @@ def post(self):
                 self.logger.exception("Exception occurred deleting an auth token")
                 abort(500, message="INTERNAL ERROR")
             else:
-                self.logger.debug("User {} logged out", user.username)
+                self.logger.warning("User {} logged out", user.username)
```

Comment:
I'd keep this as a `debug()`, informational at best.  We don't want to be too verbose as this will happen often.